channels/es.m3u

```diff
@@ -83,8 +83,10 @@ https://cls.alcarria.tv/live/alcarriatv-livestream.m3u8
 https://cartv.streaming.aranova.es/hls/live/aragonradio_aragonradio1.m3u8
 #EXTINF:-1 tvg-id="AragonTV.es" tvg-name="Aragón TV" tvg-country="ES" tvg-language="Spanish" tvg-logo="https://upload.wikimedia.org/wikipedia/commons/d/d4/Logo_aragon_tv_2016.png" group-title="",Aragón TV (720p)
 https://cartv.streaming.aranova.es/hls/live/aragontv_canal1.m3u8
-#EXTINF:-1 tvg-id="AtresSeriesInternacional.es" tvg-name="Atres Series Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atres Series Internacional
+#EXTINF:-1 tvg-id="AtreseriesInternacional.es" tvg-name="Atreseries Internacional" tvg-country="ES" tvg-language="Spanish" tvg-logo="" group-title="",Atreseries Internacional
```

Comment:
Internacional but Spanish language and Spain as a country? I'm confused

Reply:
Atreseries Internacional is a subsidiary channel of the Atresmedia Group, which is Spanish-based. That's why it's in the Spanish section.

https://www.lyngsat.com/tvchannels/es/Atreseries.html

Reply:
To be honest, the "Internacional" mention is only from the hashtag they put over their programs "#A3SeriesInt"